tools/appveyor.psm1

```diff
@@ -174,11 +181,14 @@ function Invoke-AppVeyorInstall
 {
     if(Test-DailyBuild){
         $buildName = "[Daily]"
-        if($env:APPVEYOR_PULL_REQUEST_TITLE)
+
+        # Add daily to title if it's not already there
+        # It can be there already for rerun requests
+        if($env:APPVEYOR_PULL_REQUEST_TITLE -and $env:APPVEYOR_PULL_REQUEST_TITLE  -notmatch '^\[Daily\].*$')
```

Comment:
Think that's ok... this is supposed to put [Daily] at the front... if it's not at the front, add it.